src/slider.css.ts

```diff
@@ -0,0 +1,163 @@
+import {sprinkles} from './sprinkles.css';
+import {vars} from './skins/skin-contract.css';
+import {style, styleVariants} from '@vanilla-extract/css';
+import {mq} from '.';
+
+const transition = '0.8s cubic-bezier(0.215, 0.61, 0.355, 1)';
+
+export const container = style([
+    sprinkles({
+        display: 'flex',
+        flex: 1,
+    }),
+    {
+        width: '100%',
+    },
+]);
+
+export const targetContainer = style([{display: 'inline-block', width: '100%'}]);
+
+export const rangeSlider = style([
+    sprinkles({
+        display: 'flex',
+        position: 'relative',
+        width: '100%',
+    }),
+    {},
+]);
+
+export const sliderBase = style([
+    sprinkles({
+        borderRadius: vars.borderRadii.bar,
+        background: vars.colors.control,
+    }),
+    {
+        WebkitAppearance: 'none',
+        width: '100%',
+        height: 4,
+        outline: 'none',
+        margin: 0,
+        cursor: 'pointer',
+
+        '::-webkit-slider-thumb': {
+            WebkitAppearance: 'none',
+            width: 20,
+            height: 20,
+
+            zIndex: 3,
+            position: 'relative',
+            borderRadius: vars.borderRadii.indicator,
+        },
+
+        '@media': {
+            [mq.tabletOrSmaller]: {
+                cursor: 'none',
```

Comment:
what happend if an user use a mouse or any peripheral with a cursor on a tablet?

Reply:
if the user uses this, it will continue with the expected behavior of not showing the mouse on the slider because the verification is based on the screen size

Reply:
So, why do we use cursor: none? I think we can remove this line 🤔 @atabel can you see anything that I'm missing?

Reply:
No, I don't see the point of not showing the cursor in mobile, I'd remove this line